doc/notebooks/jupyter_integration.ipynb

```diff
@@ -75,17 +76,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Let's display the DTM of our sample area to ensure all's working\n",
-    "\n",
+    "# Let's see what is in the example database\n",
+    "print(gs.read_command(\"g.list\", type=\"all\"))"
```

Comment:
I would like to see the map first and the list only later, but that might be just me.

Reply:
> I had it the other way because, if you weren't familiar with the spm dataset, seeing the contents would be a pre-requisite to knowing how to modify/play with the code below.

Reply:
Right, but I would say it is not a prerequisite to run the code. You can then executed the next cell to see what else is in the dataset and then go back and play. The other consideration is showing images versus text first. What do you want to see? This of course depends on the target audience and purpose of the notebook which I'm not sure if we decided. Promotion of the Jupyter integration is probably a good goal at this point for this notebook.

Reply:
For listing of data also:
https://grass.osgeo.org/grass79/manuals/libpython/script.html?highlight=list#script.core.list_strings
could be used...

Reply:
That's true, the list takes up a lot of space and maybe the more likely workflow is to run the whole notebook then go back and modify. I'm convinced - I'll switch it. 

@ninsbl That's a cool function! I'll try it out.

Reply:
In the introduction it is probably worth mentioning where to find more information on what one can do with GRASS GIS APIs, like:
* Manuals in general: https://grass.osgeo.org/grass-stable/manuals/
* Python API: https://grass.osgeo.org/grass-stable/manuals/libpython/
* Module manuals: https://grass.osgeo.org/grass-stable/manuals/full_index.html (or https://grass.osgeo.org/grass78/manuals/graphical_index.html)

In addition to listing data, also modules could be listed (at the end):
`print(gs.read_command("g.search.modules", flags=g"))`

Reply:
@ninsbl I was thinking this might be better placed in the example notebook, since this one is more focused in Jupyter-GRASS integration. So, I made a new PR with those changes here: https://github.com/OSGeo/grass/pull/1686

You can also see the changes in binder here:
https://mybinder.org/v2/gh/chaedri/grass/example_notebook_update?urlpath=lab%2Ftree%2Fdoc%2Fnotebooks%2Fexample_notebook.ipynb

I'm undecided on whether listing all the modules is helpful - perhaps since the manual link is there, the list is unnecessary?